modules/administration/pages/kubernetes.adoc

```diff
@@ -93,15 +101,16 @@ You might need to redeploy the image into the pod.
 . Click btn:[Create] to create an image profile.
 You will need to use a dockerfile that is suitable to deploy to {k8s}.
 . Navigate to menu:Images[Build] to build an image with the new profile.
-. Use the [command]``kubectl`` tool to deploy the image into one of the registered {k8s} clusters.
+. Deploy the image into one of the registered {k8s} clusters.
+You can do this with the [command]``kubectl`` tool.
 
 The updated data should now be available in the image list at menu:Images[Image List].
 
 
 
 // This procedure needs help. LKB 2019-10-03
 .Procedure: Importing a Previously Deployed Image
-. In the {productname} {webui}, navigate to menu:Images[Image List], and select the existing image that you want to import.
+. In the {productname} {webui}, navigate to menu:Images[Image List], and upload the existing image that you want to import.
 . Add the registry owning the image as an image store.
 . Navigate to menu:Images[Image List] and click btn:[Import].
 . Complete the fields and click btn:[Import].
```

Comment:
This part is still not right. Let's clarify. It should go like this:

 - In the SUMA UI, navigate to Images > Image Stores and add the registry from which you want to import your image, if you haven't done so before.
 - Navigate to Images > Image List, click "Import"
 - Fill in the fields, selecting the previously created "Image Store" and click "Import"

Reply:
Also for the line below, I'd suggest "imported image" instead of "updated data"